src/studio/src/designer/frontend/app-development/App.css

```diff
@@ -1,3 +1,7 @@
+body {
```

Comment:
I think the fix would instead be to [add the `AppWrapper` component](https://designsystem.altinn.studio/?path=/story/components-appwrapper--page) , which should be added when using the DS library. This will set the nescessary styles.

Reply:
The issue in app-development happened because I updated the wrong CSS file. So reverting this change and updating the correct file for the dashboard css solved the problem both places.

I tried (quickly) setting up AppWrapper around the dashboard App-component, but that did not solve the problem without the CSS update.

Reply:
Right, I didnt notice that it was the wrong file edited in the first PR either 😅

Using the AppWrapper for dashboard wont work unfortunately, because the design system has the "wrong" assumption that font-size: 62.5%/1rem = 10px (to be compatible with our old design system which has these assumptions), and that is what the AppWrapper makes sure to set up. If I remember correctly, Dashboard doesnt use any of the old design system, and so the calculated values for our new designsystem will be incorrect...

This is needed until we can get rid of the entire transformation of the REM values (so that everything is using font-size: 100%/1rem = 16px).

This whole thing is a bit confusing and unfortunate, and will likely cause some more complications down the line. Hope that we can get rid of the dependency to our old design system soon so we can remove these workarounds 🤞